recipe/meta.yaml

```diff
@@ -1,49 +1,61 @@
 {% set name = "dbt-adapters" %}
-{% set version = "1.13.0" %}
+{% set version = "1.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/dbt_adapters-{{ version }}.tar.gz
-  sha256: 2e4e743d3613e7d72319edf6eb85ea2ca57133472e32ff0148c1325f21ff6d16
+  url: https://github.com/dbt-labs/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5c0f8e34e6531b9d8787a629b7e2b63cb6eb47cc3b5d1c373a05a82fcbc9fda7
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39 or py>312]
```

Comment:
It will be clearer for future us if you add a comment about which dependency is missing for py313+